content/en/docs/setup/additional-setup/gateway/index.md

```diff
@@ -205,6 +205,10 @@ your `Gateway` selector matches these labels.
 For example, in the above deployments, the `istio=ingressgateway` label is set on the gateway pods.
 To apply a `Gateway` to these deployments, you need to select the same label:
 
+{{< tabset category-name="gateway-selector" >}}
```

Comment:
I don't think adding these 3 tabs is making this any more clear.

I would suggest to instead change line 205 to say:
```
For example, in the above deployments, the `istio=istio-ingress` label is set on the gateway pods.
```
and change the two other (not helm) deployments to match:
```
     label:
        # Set a unique label for the gateway. This is required to ensure Gateways
        # can select this workload
        istio: istio-ingress
```
This way, you don't need to explain that the helm deployment is different in the example.

Reply:
sounds good, updated.

Reply:
IIUC the helm install uses the deployment name for the label by default. If that's right, maybe a better idea is to leave the label as `istio: ingressgateway` and just change the helm command to use ingressgateway as the name. I'm not very familiar with helm, but it seems that the first argument for helm install is the name. If so, then I think that just changing the command to the following should work:
```
$ helm install ingressgateway istio/gateway -n istio-ingress
```
Then the gateway name will be the same in all 3 cases also. I wonder why it was different for helm in the first place.

Reply:
If you disagree, I've already approved, so you can remove the DNM label and merge as is.

Reply:
I test with `helm install istio-ingressgateway istio/gateway -n istio-ingress`, which looks better

```
root@ecs-zirain:~# k get po -nistio-ingress
NAME                                   READY   STATUS    RESTARTS   AGE
istio-ingressgateway-685fcc78c-jltfj   1/1     Running   0          20s
root@ecs-zirain:~# k get po -nistio-ingress --show-labels
NAME                                   READY   STATUS    RESTARTS   AGE   LABELS
istio-ingressgateway-685fcc78c-jltfj   1/1     Running   0          28s   app=istio-ingressgateway,istio.io/rev=default,istio=ingressgateway,pod-template-hash=685fcc78c,service.istio.io/canonical-name=istio-ingressgateway,service.istio.io/canonical-revision=latest,sidecar.istio.io/inject=true

```